src/scenes/end.js

```diff
@@ -1,23 +1,82 @@
-import elements from "../elements.js";
+import options from "../options.js";
 
 export default class EndScene extends Phaser.Scene {
     constructor() {
         super({ key: "EndScene" });
+        this.gameReplay = {};
+        this.buildYourOwn = {};
     }
 
     preload() {}
 
     create(data) {
-        elements.appEnd.style.visibility = "visible";
-        elements.appEnd.style.display = "flex";
-        elements.appEnd.style.marginTop = "10%";
-        elements.endScoreDisplay.innerHTML = `Score: ${data.score}`;
-
-        elements.buttonGameReplay.onclick = () => {
-            elements.appEnd.style.visibility = "invisible";
-            elements.appEnd.style.display = "none";
+        this.createDOMElements(data.score);
+
+        this.gameReplay.setInteractive();
+        this.buildYourOwn.setInteractive();
+
+        this.gameReplay.on("pointerup", () => {
             this.scene.stop("EndScene");
             this.scene.start("GameScene");
-        };
+        });
+
+        this.buildYourOwn.on("pointerup", () => {
+            window.location.href = "https://www.codecademy.com/learn/learn-phaser";
+        });
+    }
+
+    createDOMElements(score) {
+        // add score text
+        this.add.text(options.windowWidth / 2 - 110, 60, `Score: ${score}`, {
+            fontFamily: "sans-serif",
+            fontSize: "48px",
+            fontStyle: "bold",
+            color: "#000",
+            align: "center",
+        });
+
+        // add subheding
+        this.add.text(options.windowWidth / 2 - 35, options.windowHeight / 4, "So Close!", {
+            fontFamily: "sans-serif",
+            fontSize: "17px",
+            fill: "#000",
+        });
+
+        // add Play again button
+        this.gameReplay = this.add.rectangle(
+            options.windowWidth / 2,
+            options.windowHeight / 3 + 25,
+            210,
+            40,
+            0x6400e4,
```

Comment:
These hex values are reused in several places. Could you move them into `options`? As with .scss files, it's convenient to have repeated/hardcoded values stored in one place.

Also, eventually we'll want them to be able to be overridden by whoever calls `launch40Phaser`.